### DISCUSSION TITLE
Why aren't the files in the repository sorted in ascending order?

### DISCUSSION THREAD

**Discussion by SeopE9611** [community/community#140246] [Repositories] [answered]
### Select Topic Area

Question

### Body

![그림1](https://github.com/user-attachments/assets/7989a319-9254-4646-93ef-cc840df8363c)

Why aren't the files in the repository sorted in ascending order?
Also, please let me know the solution to solve this problem.

**Comment by queenofcorgis** [comment, marked as answer]
The sort of directories is in alphabetic/ lexicographic order, not numeric. 

The easiest solution for you is to prepend a leading `0` on single digit entries so 1 would be `01`, 2 `02`, etc.

**Reply by SeopE9611** [reply]
I really want to thank you for your help.

**Reply by queenofcorgis** [reply]
@SeopE9611 I'm glad I could - thanks for posting in the GitHub Community ❤️

**Comment by KavinMadusanka** [comment]
The issue you're seeing is related to the way GitHub orders files with numeric prefixes. The system doesn't handle numbers as numerical values but rather as strings, so "10" comes before "2" since it compares digits character by character.
